src/api.ts

```diff
@@ -56,6 +56,12 @@ export interface AccessoryPlugin {
    */
   identify?(): void;
 
+  /**
+   * @deprecated
+   * @internal Version needed for internal use to maintain js lecel backcompat
```

Comment:
For potential consumers, ts won't emit .d.ts for `@internal`. So it is only usable in this repo.